src/main/java/io/ebean/migration/DbNameUtil.java

```diff
@@ -16,8 +16,24 @@
   static String normalise(Connection connection) {
 
     try {
-      String productName = connection.getMetaData().getDatabaseProductName().toLowerCase();
+      final String productName = connection.getMetaData().getDatabaseProductName().toLowerCase();
       if (productName.contains(POSTGRES)) {
+        // PostgreSQL driver use a non-trustable hardcoded product name.
+        // The following block try to retrieve DBMS version to determine
+        // if used DBMS is PostgreSQL or Cockroach.
+        try {
+          final String productVersion = connection
+                  .prepareStatement("SELECT version() AS \"version\"")
```

Comment:
All looks good ... but here I think we don't close the PreparedStatement - perhaps a little adjustment here.